### PR TITLE
SEO-188026-ASP.NET-Web-Forms-DateRangePicker-display-format-H1-Missing

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Title | Description
-- | --
|Task Category | Site Audit|
|Content Task Link/Mail Screenshot | NA|
|Hotfix |https://github.com/syncfusion-content/asp.netwebforms-docs/pull/550|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187689|
|Work link |https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B0C07BBB1-1233-47A7-818C-275A98FD0063%7D&file=Faith-%20Site%20Audit.xlsx&action=default&mobileredirect=true|

Changes Made | Reason for change |
|-- | -- |
|Changed-H1 Missing | To improve performance of the page |

### CODE REVIEW

File: aspnet/DateRangePicker/display-format.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: display format
-description: display format
+title: Text representation in aspnet DateRangePicker | Syncfusion
+description: Learn here about Display Format with Syncfusion Essentialaspnet DateRangePicker control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:4]: Check title and description

File: aspnet/DateRangePicker/display-format.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 platform: aspnet
 control: DateRangePicker
 documentation: ug
 keywords: format
 ---
 
-## Display Format
+# Text representation in ASP.NET DateRangePicker Display Format
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:11]: Check the updated H1

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DateRangePicker/display-format.md (hunk 1 above)
Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md:4]: resolved

File: aspnet/DateRangePicker/display-format.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: display format
-description: display format
+title: TASP.NET DateRangePicker Display Format | Syncfusion
+description: Learn here about Display Format with Syncfusion Essentialaspnet DateRangePicker control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:4]: Recheck title and description

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DateRangePicker/display-format.md (hunk 2 above)
Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md]: comment resolved

Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md]: resolved

File: aspnet/DateRangePicker/display-format.md (hunk 3 above)
Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md]: comment resolved

File: aspnet/DateRangePicker/display-format.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: display format
-description: display format
+title: TASP.NET DateRangePicker Display Format | Syncfusion
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:3]: Check title

File: aspnet/DateRangePicker/display-format.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+description: Learn here about Display Format use in Syncfusion Essentialaspnet DateRangePicker control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:4]: Update description well

File: aspnet/DateRangePicker/display-format.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
 platform: aspnet
 control: DateRangePicker
 documentation: ug
 keywords: format
 ---
 
-## Display Format
+# ASP.NET DateRangePicker Display Format
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:11]: Update H1 properly

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DateRangePicker/display-format.md (hunk 6 above)
Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md]: comment resolved

File: aspnet/DateRangePicker/display-format.md (hunk 5 above)
Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md]: resolved

File: aspnet/DateRangePicker/display-format.md (hunk 4 above)
Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md]: resolved

File: aspnet/DateRangePicker/display-format.md [hunk 7]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: display format
-description: display format
+title: ASP.NET DateRangePicker Display Format | Syncfusion
+description: Learn here all about Display Format use in Syncfusion aspnet DateRangePicker control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:4]: Check title and description format

File: aspnet/DateRangePicker/display-format.md [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
 platform: aspnet
 control: DateRangePicker
 documentation: ug
 keywords: format
 ---
 
-## Display Format
+# Display format in ASP.NET DateRangePicker Component
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:11]: Display format in ASP.NET Web Forms DateRangePicker Component

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DateRangePicker/display-format.md (hunk 8 above)
Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md]: resolved

File: aspnet/DateRangePicker/display-format.md [hunk 9]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: display format
-description: display format
+title: Display format in ASP.NET Web Forms DateRangePicker | Syncfusion
+description: Learn here all about display Format use in Syncfusion ASP.NET Web Forms DateRangePicker control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/DateRangePicker/display-format.md:4]: Update as Learn here all about display format in Syncfusion ASP.NET Web Forms DateRangePicker control, its elements, and more.

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DateRangePicker/display-format.md (hunk 9 above)
Comment (Faith-Atieno) [reply aspnet/DateRangePicker/display-format.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**